app/src/main/res/xml/preferences_behavior.xml

```diff
@@ -2,6 +2,12 @@
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:title="@string/pref_section_behavior_title">
+    <androidx.preference.SwitchPreferenceCompat
+        android:defaultValue="false"
+        android:key="pref_focus"
```

Comment:
Let's rename this to ``pref_focus_search``.

Reply:
Sure